installer/x86_64/install.sh

```diff
@@ -559,7 +600,13 @@ $onie_menuentry
 EOF
 fi
 
-cp $grub_cfg $onie_initrd_tmp/$demo_mnt/grub/grub.cfg
+if [ "$install_env" = "build" ]; then
+    cp $grub_cfg $demo_mnt/$image_dir/platform/grub.cfg.migration
```

Comment:
Removed all changes pertaining to grub.cfg.migration since /host/machine.conf can be used instead.